### ISSUE TITLE
Change Web Console logo Alt Text

### ISSUE THREAD

**Issue by StevenBarre** [BCDevOps/developer-experience#915]
**Describe the issue**
Currently the Ansible cluster build automation is just setting the alt text to "Alt Text Here". This code should be moved into CCM and changed to the cluster name.

**Additional context**
```yaml
apiVersion: operator.openshift.io/v1
kind: Console
metadata:
  name: cluster
spec:
  route:
    hostname: "console.apps.{{ cluster_name }}.{{ basedomain }}"
  customization:
    customLogoFile:
      key: console-custom-logo.png
      name: console-custom-logo
    customProductName: KAMLOOPS SILVER CLUSTER
```

**Definition of done**
- [x] All clusters updated with new Alt Text

**Comment by StevenBarre** [comment]
https://github.com/bcgov-c/platform-gitops-gen/pull/98

**Comment by StevenBarre** [comment]
Rolled out to all clusters
